internal/integration/cli/config.go

```diff
@@ -127,7 +127,7 @@ func (suite *TalosconfigSuite) TestNew() {
 			adminOpts: []base.RunOption{base.StdoutShouldMatch(regexp.MustCompile(`MachineConfig`))},
 			readerOpts: []base.RunOption{
 				base.ShouldFail(),
-				base.StdoutEmpty(),
+				base.StdoutShouldMatch(regexp.MustCompile(`\QNODE   NAMESPACE   TYPE   ID   VERSION`)),
```

Comment:
so now we have empty tables output?

Reply:
yep, as in fact client _can_ fetch the resource definition (as it's allowed), but server denies listing actual resource